### DISCUSSION TITLE
How to create Toggle status value Hidden/Visible TextInput

### DISCUSSION THREAD

**Discussion by Fzoltan87** [filamentphp/filament#13361] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.92

### How can we help you?

How can I resolve that if "Toggle" is active, the Input field is displayed?

This doesn't work for me (but if i use checkbox it works.):
```
Toggle::make('status')
->label('Status')
->live()
->default(false),

Group::make([
    TextInput::make('text_name')->label('Text Name')->string(),
])->hidden(fn (Get $get) => $get('status')),
```

**Comment by leandrocfe** [comment, marked as answer]
```php
Forms\Components\Toggle::make('status')
    ->label('Status')
    ->live()
    ->default(false),

Forms\Components\Group::make([
    Forms\Components\TextInput::make('text_name')->label('Text Name'),
])->visible(fn (Forms\Get $get) => $get('status'))
```
?

If you are using only the Form Builder, don't forget to add `$this->form->fill()` in the mount method and add a `statePath()` to the form

https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component#adding-the-form

```php
public ?array $data = [];

public function mount(): void
{
    $this->form->fill();
}

public function form(Form $form): Form
{
    return $form
        ->schema([
            //...
        ])
        ->statePath('data');
}
```

**Reply by Fzoltan87** [reply]
I'm sorry, I only copied the part in question from the form.

This function was not specified.
```
public function mount(): void {
  $this->form->fill();
}
```

Thanks for the help.
